apps/dotcms-ui/src/app/view/components/dot-contentlet-editor/services/dot-contentlet-editor.service.ts

```diff
@@ -65,11 +67,11 @@ export class DotContentletEditorService {
         return this._header;
     }
 
-    get loadHandler(): ($event: any) => void {
+    get loadHandler(): ($event: unknown) => void {
         return this._load;
     }
 
-    get keyDownHandler(): ($event: any) => void {
+    get keyDownHandler(): ($event: unknown) => void {
```

Comment:
This is not void because we're returning something, right?

Reply:
the type of the function that returns is void no the function itself.